client/src/components/Query/Filter/Specific.jsx

```diff
@@ -260,13 +260,18 @@ class SpecificFilter extends Filter {
   }
 
   render() {
-    const { allOptions } = this.state;
+    const { allOptions, draft } = this.state;
+    const { config } = this.props;
+    const { operand } = draft;
+
     return (
       <Filter
         {...this.props}
+        config={config}
         type={FILTER_TYPE_SPECIFIC}
         editor={this.getEditor()}
         searchable
+        operand={operand}
```

Comment:
@FrancisRenaud No need to pass `operand` props since it is already contained within the data prop under `data.operand` for the relevant operators (should ba available in the draft too).